src/theme/theme.ts

```diff
@@ -260,3 +267,31 @@ export default createTheme({
     },
   },
 });
+
+theme = createTheme(theme, {
+  typography: {
+    heading: {
+      fontFamily: "Raleway",
+      fontSize: theme.typography.xxl.fontSize,
```

Comment:
I needed to separate this part of the typography in order for me to be able to reuse the `theme` properties